static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -255,10 +225,10 @@ function FlamegraphZoomView({
             1
           ),
           {
-            BORDER_COLOR: flamegraphRenderer.theme.COLORS.SELECTED_FRAME_BORDER_COLOR,
-            BORDER_WIDTH: flamegraphRenderer.theme.SIZES.FRAME_BORDER_WIDTH,
+            BORDER_COLOR: flamegraphTheme.COLORS.SELECTED_FRAME_BORDER_COLOR,
+            BORDER_WIDTH: flamegraphTheme.SIZES.FRAME_BORDER_WIDTH,
           },
-          flamegraphRenderer.configViewToPhysicalSpace
+          view.fromConfigView(canvas.physicalSpace)
```

Comment:
calling fromConfigView with physical space seems a bit odd

Reply:
As discussed in slack, gonna leave this for now until we think of a better name